libr/core/cmd_anal.c

```diff
@@ -4116,21 +4116,21 @@ static bool anal_fcn_data (RCore *core, const char *input) {
 		int i;
 		bool gap = false;
 		ut64 gap_addr = UT64_MAX;
-		char *bitmap = calloc (1, fcn->size);
+		char *bitmap = calloc (1, r_anal_fcn_size (fcn));
 		if (bitmap) {
 			RAnalBlock *b;
 			RListIter *iter;
 			r_list_foreach (fcn->bbs, iter, b) {
 				int f = b->addr - fcn->addr;
-				int t = R_MIN (f + b->size, fcn->size);
+				int t = R_MIN (f + b->size, r_anal_fcn_size (fcn));
```

Comment:
Yeah right. I thought that the compiler would be able to inline this considering it's so simple, but it doesn't even with -O3. The only way I was able to make it inline this function was by putting it in r_anal.h with static + inline. But for the moment it's probably just better to use the function in a better way. So I will review again my patch and avoid calling the function multiple times when possible. Do you agree?